### ISSUE TITLE
xi claims that he can see no difference regarding examples demonstrating the deficiency of WCAG2 vs the demonstrated efficacy of APCA

### ISSUE THREAD

**Issue by Myndex** [xi/apca-introduction#6]
Extension of #2 whcih you closed before I could comment.

> _The images you posted are far from obvious evidence, at least for me. I am not sure why you treat them as such._

So then you are claiming that, in the following two samples, the one on the LEFT is equally readable as the one on the RIGHT:

<img height="55" alt="WCAG 3:1 examples" src="https://user-images.githubusercontent.com/42009457/181590676-c8287801-824b-4c7a-bc89-f68b9f510f68.jpeg">

And you are therefore further claiming that, in the following two samples, one of these is _substantially_ less readable

<img height="55"  alt="APCA 45 examples"  src="https://user-images.githubusercontent.com/42009457/181590689-aeb5d7c2-c36f-4343-a5eb-e4e3ac525627.jpg">

Before I go further, I want to clarify that you are claiming that you see no difference between the top two samples, and the second set of samples. Because that is what you stated in issue #2 that you closed before I could comment.

I look forward to your response.

**Comment by xi** [comment]
You can still comment on closed issues, and if necessary I can reopen issues on request. But I think in this case it is a good thing to split this out into a separate issue so we can have a more focused discussion.

I also think that "Addressing claims" was too general a title for this issue, so I changed it. I hope you agree with the new title, otherwise feel free to change it to something more fitting.

 > So then you are claiming that, in the following two samples, the one on the LEFT is equally readable as the one on the RIGHT:
> And you are therefore further claiming that, in the following two samples, one of these is substantially less readable

Honestly, I am not sure. The difference between the two samples is certainly not "obvious" to me.

But that is not really the point. Showing a single word in a single color to a single person under a single set of viewing conditions fundamentally cannot be "obvious evidence". So even if I personally, in my current viewing conditions, would see a significant difference between the samples that would still not convince me. There needs to be a larger study, including people with different vision deficiencies (see https://github.com/w3c/silver/issues/574).

Maybe, if WCAG was completely and obviously wrong, a few such samples would be sufficient proof that APCA is better. But WCAG really is ok. It has been used for years and did its job. I too believe that it has issues and that APCA might be better. But, since WCAG is already fine, proving that APCA is actually better is much harder than showing a few images.

**Comment by xi** [comment]
I tried to extract the colors from those images. They use lossy jpg encoding, but I think it is something like the following (@Myndex feel free to correct me):

| background | foreground | WCAG2 | APCA |
| -- | -- | -- | -- |
| `#595959` | `#000000` |  3.0 | 20 |
| `#ffffff` | `#969696` | 3.0 | 56 |
| `#909090` | `#000000` | 6.6 | 45 |
| `#eeeeee` | `#999999` | 2.5 | 45 |

**Comment by xi** [comment]
So… there is no issue here. The title is "xi claims he [sic] can see no difference" and that is exactly true. @Myndex cannot decide whether I see a difference or not.

**Comment by Myndex** [comment]
> So… there is no issue here. The title is "xi claims he [sic] can see no difference" and that is exactly true. @Myndex cannot decide whether I see a difference or not.

It is disingenuous to claim that you can not, in other words, I do not believe you.

**Comment by Myndex** [comment]
> _The title is "xi claims he [sic] can see no difference" and_ 

COMPLETELY UNRELATED: why did you place [sic] in your quote? Are you identifying as female? Tobias is generally a male name, and you have no indication that you identify as female anywhere I can see. If you prefer a different pronoun please state so implicitly.
